src/scripts/main.js

```diff
@@ -1,3 +1,63 @@
 'use strict';
 
-// write code here
+const tbody = document.querySelector('tbody');
+const row = tbody.querySelectorAll('tr');
+
+[...row].forEach(trtb => {
+  trtb.children[0].className = 'name';
+  trtb.children[1].className = 'position';
+  trtb.children[2].className = 'age';
+  trtb.children[3].className = 'salary';
+});
+
+const thead = document.querySelector('thead');
+const tr = thead.querySelectorAll('tr');
+
+[...tr].forEach(trth => {
+  trth.children[0].id = 'name';
+  trth.children[1].id = 'position';
+  trth.children[2].id = 'age';
+  trth.children[3].id = 'salary';
+});
+
+thead.addEventListener('click', (event) => {
+  let sortTable;
+
+  if (event.target === document.querySelector('#salary')) {
+    sortTable = [...row].sort((a, b) => {
+      const aSalary = a.children[3].innerText.replace(/[^0-9]/g, '');
+      const bSalary = b.children[3].innerText.replace(/[^0-9]/g, '');
+
+      return aSalary - bSalary;
+    });
+  }
+
+  if (event.target === document.querySelector('#age')) {
+    sortTable = [...row].sort((a, b) => {
+      const aAge = +a.children[2].innerText;
+      const bAge = +b.children[2].innerText;
+
+      return aAge - bAge;
+    });
+  }
+
+  if (event.target === document.querySelector('#position')) {
```

Comment:
you can get data attribute from event.target and compare it with string

Reply:
it has no data attribute. should I add it?